flink-dist/src/main/flink-bin/bin/config.sh

```diff
@@ -249,7 +249,15 @@ if [ -n "$HADOOP_HOME" ]; then
     fi
 fi
 
-INTERNAL_HADOOP_CLASSPATHS="$HADOOP_CLASSPATH:$HADOOP_CONF_DIR:$YARN_CONF_DIR"
+INTERNAL_HADOOP_CLASSPATHS="${HADOOP_CLASSPATH}:${HADOOP_CONF_DIR}:${YARN_CONF_DIR}"
+
+if [ -n "${HBASE_CONF_DIR}" ]; then
+    # Setup the HBase classpath.
+    INTERNAL_HADOOP_CLASSPATHS="${INTERNAL_HADOOP_CLASSPATHS}:`hbase classpath`"
```

Comment:
1) You see an error during the startup of your flink job
2) An empty value is inserted in the path causing your homedirectory to be in the classpath.
3) HBase support doesn't work.

A quick test on the commandline:
```
$ INTERNAL_HADOOP_CLASSPATHS=foo:bar
$ INTERNAL_HADOOP_CLASSPATHS="${INTERNAL_HADOOP_CLASSPATHS}:`hbasexx classpath`"
bash: hbasexx: command not found...
$ echo $INTERNAL_HADOOP_CLASSPATHS 
foo:bar:
```